src/content/tutorials/fleet-management/app-platform/app-bundle/index.md

```diff
@@ -22,80 +22,81 @@ last_review_date: 2024-10-28
 
-As stated in the [app platform overview]({{< relref "/overview/fleet-management/app-management" >}}) all managed apps are packaged, maintained and offered as `helm` charts and it's no different for the app bundles. What makes them special in comparison to solitary apps is that the bundle `helm` chart, instead of carrying a regular resources composing the actual application, carries [`App` custom resources]({{< relref "/reference/platform-api/crd/apps.application.giantswarm.io.md" >}}) which, once delivered and consumed by the app platform, install the expected applications and their resources.
+As stated in the [app platform overview]({{< relref "/overview/fleet-management/app-management" >}}), all managed apps are packaged, maintained, and offered as `helm` charts, and it's no different for app bundles. What makes them special compared to solitary apps is that the bundle `helm` chart, instead of carrying regular resources composing the actual application, carries [`App` custom resources]({{< relref "/reference/platform-api/crd/apps.application.giantswarm.io.md" >}}) which, once delivered and consumed by the app platform, install the expected applications and their resources.
 
-In other words, an app bundle can be thought of a meta package, not installing anything in the workload cluster on its own, but requesting installation of certain pre-defined applications.
+In other words, an app bundle can be thought of as a meta package. It doesn't install anything in the workload cluster on its own, but instead requests installation of certain pre-defined applications.
 
-The distinction between solitary (on the left side) and bundle (on the right side) apps is depicted in the
-figure below.
+The distinction between solitary and bundle apps is depicted in the figure below.
 
 ![Solitary app helm chart vs Bundle app helm chart](app-bundle-vs-normal-app.png)
 
-The `A1` apps `helm` chart consists of end resources representing the actual application. Upon installing this app, these resource are created in the **workload cluster directly**. On the other side is a bundle `helm` chart carrying the `A1-A4` `App` custom resources. Installing this bundle results in the creation of these nested `App` custom resources in the **management cluster**. These `App` custom resources are then picked up and reconciled by the app platform which only then results in creating end resources in the workload cluster.
+The `A1` app's `helm` chart consists of end resources representing the actual application. Upon installing this app, these resources are created in the **workload cluster directly**. On the other side is a bundle `helm` chart carrying the `A1-A4` `App` custom resources. Installing this bundle results in the creation of these nested `App` custom resources in the **management cluster**. These `App` custom resources are then picked up and reconciled by the app platform, which only then results in creating end resources in the workload cluster.
```

Comment:
"end resources" sounds weird to me. Can the be replaced with something like final resources?

Reply:
`children resources` maybe?